cpp/harvest/nacjd_pipeline.sh

```diff
@@ -11,15 +11,16 @@ declare -r BSZ_FILENAME="krimdok_nacjd_$(date +%y%m%d)_001"
 declare -r DOWNLOAD_DIR=/tmp/NACJD/$(date +%Y%m%d_%H%M%S)
 declare -r DOWNLOAD_FILE="$DOWNLOAD_DIR/$BSZ_FILENAME.json"
 declare -r CONVERT_FILE="$DOWNLOAD_DIR/$BSZ_FILENAME.xml"
+declare -r BSZ_DIR=/usr/local/ub_tools/bsz_daten
 mkdir -p "$DOWNLOAD_DIR"
 
 # Use empty MRC file as dummy input.
 # On consecutive downloads we want to use the real KrimDok GesamtTiteldaten... file here.
-declare -r EMPTY_MRC_FILE="$DOWNLOAD_DIR/empty.mrc"
-touch "$EMPTY_MRC_FILE"
+declare -r GESAMTTITLE_DATEN = "$BSZ_DIR/GesamtTitelDaten-$(date +%y%m%d).mrc"
```

Comment:
- A better variable name would be `GESAMTTITELDATEN_FILE` or `REFERENCE_FILE`.
- You should move the declaration up next to the other declarations.